.github/workflows/prod.yml

```
@@ -8,28 +8,41 @@ on:
+          cd techeerzip
+          ./gradlew test jacocoTestReport # 단위 테스트
 
+      - name: Check coverage
+        continue-on-error: true
+        uses: madrapps/jacoco-report@v1.7.2
+        with:
+          paths: |
+            ${{ github.workspace }}/techeerzip/build/reports/jacoco/test/html/index.html
+            ${{ github.workspace }}/techeerzip/build/reports/jacoco/test/jacocoTestReport.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-coverage-overall: 30  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)